src/index.html

```diff
@@ -7,8 +7,84 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div>
+        <a class="logo" href="#home">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__items">
+            <a class="is-active"
+                rel="liner"
+                href="#apples"
+            >
```

Comment:
```suggestion
            <a 
              class="is-active"
              rel="liner"
              href="#apples"
            >
```

check your code style

Reply:
fix all such cases

Reply:
as it was shown in the previous comments
<img width="358" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/102411130/d0708f5e-668e-49f0-b4a0-d4430e07bd57">